cdap-e2e-tests/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -3,3 +3,16 @@ clientUrl=http://localhost:11011
 serverUrl=https://placeholder.com/api
 # command to generate token: gcloud auth print-access-token
 serverAccessToken=placeholder
+projectId=cdf-athena
+
+## HUB-PROPERTIES-START
+expectedPluginName=Anaplan
+expectedElement=HUB
+invalidMessage_hub=sfts
+Artifacts=Artifacts-input
+gcsSourceBucket=dummy
+gcsTargetBucketName=dummy
+gcsTargetPath=dummy
+testFile=test.csv
```

Comment:
Provide the proper absolute file path after placing it in testdata folder and use it.

Reply:
done